share/completions/composer.fish

```diff
@@ -20,12 +20,26 @@ function __fish_composer_using_command
   return 1
 end
 
+function __fish_composer_packages
+    echo "
+import json
+json_data = open('composer.json')
```

Comment:
Is it safe to assume *composer.json* is in the CWD?

Using python for this is fine since fish has no ability to handle JSON (this might change in the 3.x releases planned for next year).